src/LanguageServer.ts

```diff
@@ -1214,6 +1214,7 @@ export class LanguageServer {
 
             const activeSignature = signatures.length > 0 ? 0 : null;
 
+            // TODO: confirm whether this is intentional. `null >= 0` is `true`.
```

Comment:
This seems likely to be a bug to me

Reply:
I fixed and added [a test](https://github.com/rokucommunity/brighterscript/pull/996/files#diff-433a120cab1aa0a63fe01648a9a73f0bc29ad7f5981d92838ec863cdff2d8244R591-R604). I don't have a lot of context here, but it was possible to trigger the condition by invoking `onSignatureHelp` on an XML file. Once I had the test in place I saw that the "bug" wasn't anything scary, and that it's just a field being set to `undefined` when it should have been set to `null`.